### PR TITLE
Option to disable drawing of active window border

### PR DESCRIPTION
**Pull request by akhilles**
`draw_active_window_borders` option added, defaults to True. Unaffected by draw_minimal_borders and window_margin_width.

### CODE REVIEW

File: kitty/layout.py [hunk 1]
```diff
@@ -26,8 +27,9 @@ def idx_for_id(win_id, windows):
 
 
 def set_draw_minimal_borders(opts):
```
Comment (akhilles) [inline kitty/layout.py]: Maybe this function name can be generalized to set_draw_border_flags()?

**Comment by kovidgoyal** [comment]
I dont understand how this is supposed to work. As far as I cant tell, it will simply draw the borders around the active window in the inactive border color instead of the active border color. If that is what you want to do, why not just set the active border color to be the same as the inactive border color in kitty.conf?

**Comment by akhilles** [comment]
This is to completely disable rendering of active window border. If I change the inactive border color to active border color, it still renders the extra edges as you switch active window.

You said I could submit a PR for this, but then just rejected it .... :(

**Comment by akhilles** [comment]
Adding a screenshot:
image

**Comment by kovidgoyal** [comment]
Oh sorry my mistake, I missed a part of the patch.

**Pull request closed without merging**